frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -276,10 +296,16 @@
 
         <MoreSettings>
           <button class="btn btn-error" on:click={softDeleteProject}>
-            {$t('delete_project_modal.submit')}<TrashIcon />
           </button>
         </MoreSettings>
       {/if}
+      <AdminContent>
+        <p class="text-2xl mb-4">
+          <button class="btn btn-accent" on:click={() => resetProject()}>
```

Comment:
If we ever change the `canManage` rules to remove `isAdmin` then it might possibly cause issues (admin suddenly can't see the reset project button), but that's very unlikely. I'll do it, because that certainly will look visually better.